__tests__/app.test.js

```diff
@@ -29,34 +29,72 @@ describe("GET: /api", () => {
       .get("/api")
       .expect(200)
       .then(({ body }) => {
-        expect(body.endpoints).toEqual(endpoints);
+        const testEndPoints = body.endpoints;
+        expect(testEndPoints).toEqual(endpoints);
         expect(typeof body.endpoints).toBe("object");
       });
   });
 });
 
-describe("GET: /api/articles/:article_id", () => {
-  test("GET: 200 - returns an article based on the article_id given by user", () => {
+describe("GET: 200 /api/articles/:article_id", () => {
+  test("Should return the details for article 1 as that is the endpoint being navigated to", () => {
     return request(app)
       .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
-        expect(body.article_id.article_id).toBe(1);
-        expect(typeof body.article_id.title).toBe("string");
-        expect(typeof body.article_id.topic).toBe("string");
-        expect(typeof body.article_id.author).toBe("string");
-        expect(typeof body.article_id.body).toBe("string");
-        expect(typeof body.article_id.created_at).toBe("string");
-        expect(typeof body.article_id.votes).toBe("number");
-        expect(typeof body.article_id.article_img_url).toBe("string");
+        expect(body.article.article_id).toBe(1);
+        expect(typeof body.article.title).toBe("string");
+        expect(body.article.title).toBe("Living in the shadow of a great man");
+        expect(typeof body.article.topic).toBe("string");
+        expect(body.article.topic).toBe("mitch");
+        expect(typeof body.article.author).toBe("string");
+        expect(body.article.author).toBe("butter_bridge");
+        expect(typeof body.article.body).toBe("string");
+        expect(body.article.body).toBe("I find this existence challenging");
+        expect(typeof body.article.created_at).toBe("string");
+        expect(typeof body.article.votes).toBe("number");
+        expect(body.article.votes).toBe(100);
+        expect(typeof body.article.article_img_url).toBe("string");
+        expect(body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
```

Comment:
You might want to look at the jest matcher `toHaveProperty` and `expect.any` they can be really handy for this sort of task